config/internal/webhook/validating_webhook.yaml.tmpl

```
@@ -0,0 +1,34 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: pipelineversions.pipelines.kubeflow.org
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+    {{ if .APIServer.WebhookAnnotations }}
+    {{ range $key, $value := .APIServer.WebhookAnnotations }}
+    {{ $key }}: "{{ $value }}"
```

I think this may be safer.
```suggestion
    {{ printf "%q" $key }}: {{ printf "%q" $value }}
```